media/open_zim.ksy

```diff
@@ -0,0 +1,380 @@
+meta:
+  id: open_zim
+  title: "(Open) Zeno IMproved"
+  application:
+    - Kiwix
+    - libzim
+    - zimlib
+  file-extension: zim
+  xref:
+    wikidata: Q784695
+  license: CC-BY-SA-3.0
+  encoding: utf-8
+  endian: le
+  -requires3rdParty:
+    - https://github.com/kaitai-io/kaitai_compress
```

Comment:
OK, replaced with `-requires-optional`

Reply:
>And why do you put this information into a special invented key and don't you just write it into the doc?

Because structured is better than non-structured. We can create a convention which keys mean what, and it is easier to read such docs than a free-form text. And it is easier to process with scripts, when it is needed. So more such keys - the better, if we keep tyem consistent and their names human-understandable.

>This key does not have any potential to become machine-readable anyway, and by attempting to make it so (and not just describing the info in a free-form text field) you only tie your hands yourself because you can't express anything else than the URL 

It doesn't tie my hands: URIs can be distinguished from free-form text automatically.

>(you can't say anything like why this KSY spec depends on it, what happens if you don't install it, some pitfalls, etc.).

It xcan be said in commsnts.